authz/src/main/java/io/grpc/authz/AuthorizationServerInterceptor.java

```diff
@@ -31,14 +31,20 @@
 import java.util.List;
 
 /**
- * Authorization server interceptor for static policy.
+ * Authorization server interceptor for static policy. The class will get
+ * gRPC Authorization policy as a JSON string during initialization. This
```

Comment:
Have "gRPC Authorization policy" be a link to https://github.com/grpc/proposal/blob/master/A43-grpc-authorization-api.md#user-facing-authorization-policy

Reply:
Done